torchci/lib/benchmark/aoUtils.ts

```diff
@@ -1,5 +1,15 @@
+import { LLMsBenchmarkData } from "components/benchmark/llms/common";
 import { BenchmarkData, CompilerPerformanceData } from "lib/types";
 
+export const TORCHAO_REPO = "pytorch/ao";
+// TODO (huydhn): Find a better way to abstract this baseline concept, for example,
+// this could be dtype noquant for TorchAO, or eager config for inductor
+export const TORCHAO_BASELINE = "noquant";
+// TODO (huydhn): The following are TorchAO speedup metrics. Check with ao team to
+// see if this information could be codified on the benchmark instead of keeping it
+// here on the dashboard
+const SPEEDUP_METRICS = ["tok/s", "time_ms(avg)", "time_s(avg)", "img_s(avg)"];
```

Comment:
you mean give a more unified metric?

Reply:
That or having a field in the benchmark record like `require_speedup=True` are both ok I think.  On the other hand, we don't even need this anymore if TorchAO can upload the speedup metric directly.  I will think more about the pros and cons of these 2 approaches